hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/TestMiniChaosOzoneCluster.java

```diff
@@ -136,7 +136,7 @@ public static void init() throws Exception {
     cluster.waitForClusterToBeReady();
 
     String volumeName = RandomStringUtils.randomAlphabetic(10).toLowerCase();
-    ObjectStore store = cluster.getRpcClient().getObjectStore();
+    ObjectStore store = cluster.newClient().getObjectStore();
```

Comment:
Do we want to close this created client?

Reply:
Thanks, I missed this, was only checking `integration-test`.